src/loader.js

```diff
@@ -1,102 +1,97 @@
-const loaderUtils = require("loader-utils");
+const loaderUtils = require('loader-utils')
```

Comment:
One general question. There are many formatting changes. Did you run `npm run format` before commit?

Reply:
I think I've never run prettier after renovate bumped its version. And you run it :) That's why so much changes. But that's fine, I was just curious if it was prettier or IDE formatting.

Reply:
😄 I tried also running format with my VSCode and returned the same output. Single quote is the new way to go 👍🏻